Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -671,9 +683,18 @@ script:
     - contextPath: Gmail.Headers
       description: All headers of specific mail (list).
       type: unknown
-    - contextPath: Gmail.Mailbox
-      description: The Gmail Mailbox.
-      type: string
+    - contextPath: Gmail.Mailboxes
+      description: The Gmail mailboxes in which emails matching the query were found.
+      type: String
+    - contextPath: Gmail.UnsearchedAcounts
+      description: The unsearched acounts with the error.
+      type: String
+    - contextPath: PageToken.NextPageToken
+      description: The use of this output is in the Search all mailboxes - Gmail with polling Playbook, so NextPageToken is used to call back the command to fetch the list of the next accounts in which the search will be made.
+      type: String
+    - contextPath: SearchProcess
+      description: The use of this output is in the Search all mailboxes - Gmail with polling Playbook, it allows the search process to be followed.
```

Comment:
Updated this to "Enables the search process to be followed." - is this ok?

Reply:
yes